scripts/live-connector-e2e/run-windows.ps1

```
@@ -1191,12 +1215,13 @@ function Invoke-Teardown {
 function Invoke-Hook([string]$EventName, [string]$Payload, [ValidateSet('allow', 'block')][string]$Expected, [bool]$RequireGatewayBlock = $false) {
     $before = @(Get-EventLines $script:GatewayJsonl).Count
     $result = Invoke-Tool (Resolve-ContractHookTool) @('hook', '--connector', $Connector, '--event', $EventName) @(0, 2) -InputPath $Payload
-    Start-Sleep -Milliseconds 800
-    if (-not (Test-ConnectorEvent $script:GatewayJsonl $Connector $before)) { throw "$EventName did not reach the gateway" }
+    $requireBlockEvidence = $Expected -eq 'block' -or $RequireGatewayBlock
+    $evidence = Wait-GatewayEvidenceAfter $script:GatewayJsonl $Connector $before $requireBlockEvidence
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Wait-GatewayEvidenceAfter' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)